spec/factories/invoice_items.rb

```diff
@@ -8,8 +8,8 @@
   end
 
   factory :invoice_item do
-    invoice_id 1
-    item_id 1
+    invoice
+    item
     quantity
```

Comment:
Thanks for doing this. I've definitely not been using factory_girl to it's full potential.